docs/Usage.md

```diff
@@ -94,7 +94,63 @@ The exported ONNX file may contain special, non-ONNX-standard operations and lay
 In some cases it is possible to export a compressed model with ONNX standard operations only (so that it can be run using `onnxruntime`, for example) - this is the case for the 8-bit symmetric quantization and sparsity/filter pruning algorithms.
 Refer to [compression algorithm documentation](./compression_algorithms) for details.
 
-## Saving and loading compressed models in PyTorch
+## Saving and loading compressed models
+The complete information about compression is defined by a compressed model and a compression state.
+The model characterizes the weights and topology of the network. The compression state - how to restore the setting of 
```

Comment:
```suggestion
The compressed model characterizes the weights and topology of the network. The compression state - how to restore the setting of 
```

Reply:
@daniil-lyakhov, IMHO, original sentence has the same meaning thanks to the article `the`

Reply:
Ok, but the compression state `compression` word is not omitted with article `the`, this is a bit misleading for me